quinn-proto/src/connection/paths.rs

```diff
@@ -67,20 +74,15 @@ impl PathData {
         PathData {
             remote,
             rtt: prev.rtt,
-            pacing: Pacer::new(
-                smoothed_rtt,
-                congestion.window(),
-                prev.max_udp_payload_size,
-                now,
-            ),
+            pacing: Pacer::new(smoothed_rtt, congestion.window(), prev.current_mtu(), now),
```

Comment:
Do we need to update the pacer when the MTU changes?

Reply:
I think we do. The pacer's capacity is derived from the rtt and the congestion window, but it might end up being a different value because it is clamped between `10 * MTU` and `256 * MTU`. Before this PR, the capacity was only recalculated when the size of the congestion window changes, but it is clear that it should also be recalculated when the MTU changes (I just pushed a commit to that effect). Does that make sense?

Reply:
LGTM.